modules/moa-installing-prerequisites.adoc

```diff
@@ -161,11 +161,11 @@ After both the permissions and quota checks pass, proceed to preparing your AWS
 +
 . Prepare your AWS account for cluster deployment
 +
-.. Run the following command to log in to your Red Hat account with `moactl`.
+.. Run the following command to log in to your Red Hat account with `rosa`.
 +
 [source,terminal]
 ----
-$ moactl login
+$ rosa login
 ----
 +
 Replace `<my_offline_access_token>` with your token.
```

Comment:
In line 176 below, the URL https://cloud.redhat.com/openshift/token/moa still works but redirects to https://cloud.redhat.com/openshift/token/rosa. May want to update.